### DISCUSSION TITLE
Guides

### DISCUSSION THREAD

**Discussion by okuramasafumi** [okuramasafumi/alba#114] [General]
I'm adding a document which guides users to do various things with Alba. I'd love to see what the guide should mention from the users' perspective.
For example, the guide might tell how to skip defining classes for all resources but instead use blocks to define associations.

**Comment by tashirosota** [comment]
I want how to switch from another serializer.
This example helps in adding Alba to project, and we easily can get high speed serialization.

**Comment by okuramasafumi** [comment]
#98 is the issue for the document describing how to switch from other serializers.
